AppHandling/Run-AlPipeline.ps1

```diff
@@ -485,6 +485,11 @@ if ($sharedFolder -and !(Test-Path $sharedFolder -PathType Container)) {
     throw "If sharedFolder is specified, it must be an existing folder"
 }
 
+if($keepContainer -and !$credential) {

```

Comment:
Please also remove lines 670 - 678 (the dump on whether credentials is specified) - there is no reason to dump this info anymore.

Reply:
Done.